src/constants.js

```diff
@@ -49,3 +49,10 @@ export const DECODED_ROUTES = {
     '/container/:blockId',
   ],
 };
+
+export const COURSE_BLOCK_NAMES = /** @type {const} */ ({
```

Comment:
```suggestion
export const COURSE_BLOCK_NAMES = ({
```

I'm all for JSDoc, but I'm not sure I like having it sprinkled piecemeal in a file.  If we go for it, we should do it for all instances.  (At least on a per-file basis.)

Reply:
Completely agree with you.
JSDoc was deleted for this case and resolved conflicts.